laravel/resources/views/banners.blade.php

```diff
@@ -53,7 +53,7 @@
     <div class="row">
         <div class="col-lg-12">
             <div class="alert alert-primary" role="alert">
-                There are no Banners configured! <a href="#addBanner" data-bs-toggle="modal" data-bs-target="#addBanner">Add now a new Banner</a>
+                There are no Banners configured! <button class="btn btn-link p-0" type="button" data-bs-toggle="modal" data-bs-target="#addBanner">Add now a new Banner</button>
```

Comment:
[CONCERN]

The `@can()` directive is missing here.

Reply:
the can directive is above the row
```
<div class="container">
    @can('add banners')
    <div class="row">
        <div class="col-lg-3">
            <button type="button" class="btn btn-primary btn-sm fw-bold" data-bs-toggle="modal" data-bs-target="#addBanner">
                Add Banner
            </button>
        </div>
    </div>
    <hr>
    @endcan
    @include('inc.standard-alerts')
</div>
```